app/controllers/photos_controller.rb

```diff
@@ -1,5 +1,6 @@
 class PhotosController < ApplicationController
   before_action :set_photo, only: %i[ show edit update destroy ]
+  bwfore_actoin :ensure_current_user_is_owner, only: [:destroy, :update, :edit]
```

Comment:
Misspelled

Reply:
I never pushed the update that fixed this, my bad. fixed now hopefully